.github/workflows/java.yml

```diff
@@ -197,7 +197,8 @@ jobs:
               name: lint java rust
 
     test-modules:
-        if: github.repository_owner == 'valkey-io'
+        if: github.event.pull_request.head.repo.owner.login == 'valkey-io'
```

Comment:
CI started on demand (`workflow_dispatch`) doesn't run modules test.
Please, redo the fix.